### PR TITLE
Add skip layer normalization

### PR DESCRIPTION
**Pull request by TedThemistokleous**
Needed to support customer models. Seen in optimized versions of BERT or optimizations using the Onnxruntime toolset

Currently built off the add_attention_contrib_op branch as this is used to parse in an optimized bert model

official Doc here: https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#com.microsoft.SkipLayerNormalization

Other useful descriptions (this vs say simplified case)

https://pytorch.org/docs/stable/generated/torch.nn.RMSNorm.html#torch.nn.RMSNorm (also known as simplified)

https://sh-tsang.medium.com/review-layer-normalization-ln-6c2ae88bae47

### CODE REVIEW

**Comment by TedThemistokleous** [comment]
Moved to 3917 to get that into develop by @ahsan-ca
